app/frontend/packs/ielts-band-score-autocomplete.js

```diff
@@ -19,6 +19,9 @@ const initIeltsBandScoreAutocomplete = () => {
         showNoOptionsFound: true,
         confirmOnBlur: false
       });
+
+      const accessibleAutocompleteWrapper = document.querySelector(".autocomplete__wrapper");
```

Comment:
I don't think it's an issue on the page where it's used but should/could this selector be more specific to avoid picking up other autocompletes (maybe in the future)?

Reply:
@stevehook Probably not a blocker, but have updated to add a class to the parent form group, and made this a more specific selector.